common/src/primitives.rs

```diff
@@ -179,6 +176,24 @@ pub const DEMETER_ASSET_ID: AssetId32<PredefinedAssetId> = AssetId32::from_bytes
 pub const HERMES_ASSET_ID: AssetId32<PredefinedAssetId> = AssetId32::from_bytes(hex!(
     "002d4e9e03f192cc33b128319a049f353db98fbf4d98f717fd0b7f66a0462142"
 ));
+#[cfg(not(feature = "private-net"))]
+pub const DOT: AssetId32<PredefinedAssetId> = AssetId32::from_bytes(hex!(
+    "0003b1dbee890acfb1b3bc12d1bb3b4295f52755423f84d1751b2545cebf000b"
+));
+#[cfg(not(feature = "private-net"))]
+pub const KSM: AssetId32<PredefinedAssetId> = AssetId32::from_bytes(hex!(
```

Comment:
Does it work for tests without `private-net` feature. We use KSM and DOT for tests in some pallets

Reply:
Thanks for pointing out! Just ran the test without `private-net`, and they pass successfully on my machine.
I suspect the tests use this const instead of hardcoded value, thus making them id-agnostic; everything seems fine :)